ttnn/cpp/ttnn/operations/loss/loss.hpp

```diff
@@ -0,0 +1,69 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <functional>
+#include <optional>
+
+#include "loss_types.hpp"
+#include "ttnn/tensor/tensor.hpp"
+#include "ttnn/run_operation.hpp"
+#include "ttnn/decorators.hpp"
+#include "ttnn/operations/core/core.hpp"
```

Comment:
among these includes we likely only need `#include "ttnn/decorators.hpp"`

Reply:
done.

Reply:
Thank you!